### PR TITLE
Fix/create card banner fix dropdown styles and router issue

### LINKED ISSUE #696: Improve router usage after a group is created.
### Problem

Context: https://github.com/okTurtles/group-income-simple/pull/679/files#r329122719

There was a weird rare false positive in a test failure in this step. Let me explain:

image

This view is visible for 1ms. It happens immediately after a user with a group logs in. How is that possible? Well...
Let's see the current conditions for each view:
- sidebar: isloggedIn and hasGroups
- welcomeGroup: isloggedIn

So, both conditions are indeed true.

If we look at the code, after the login happens, there's a tiny moment where we verify it the user has a group and then ask $router to redirect to `/dashboard`.
```js
// Login.vue#L91
await sbp('state/vuex/dispatch', 'login', { username, identityContractID })
if (this.$store.state.currentGroupId) {
  this.$router.push({ path: '/dashboard' })
}
```
That's the exactly moment where this interface (the screenshot) takes place in time.

For that reason, the condition to find "welcomeHomeLoggedIn" (L33) is positive. However, in the next millisecond, when we try to get the element again and assert the text, the element isn't there anymore because the $router has changed to `/dashboard`.

Based on this, it's a tricky task to get the correct element on the page in this moment. So I decided to change the assertions: if the login was successful, it means we won't display the login button anymore. That's it.

I think the ideal solution would be to move the welcome view (create/join group) to another router (ex: `/welcome`).  That would prevent this bug and we would still have better login assertions.

### Solution

Improve how we are using the router `/welcome` and how we present that to the users after a group is created.

cc @pieer and @taoeffect

### PR DESCRIPTION
**Pull request by sandrina-p**
- Create card banner component (cc @mmbotelho )
- Fix dropdown styles  (cc @mmbotelho )
- Add reduced-motion feature (enabled by default on Cypress) and try to fix #696

### CODE REVIEW

**Review by taoeffect — COMMENTED** [review]
This looks good! Only concern I have at the moment is whether "Banner" is a good name for the new component you created, since @mmbotelho has designed something called "Banners" and already is using them for a very different purpose here: #688.

To me the "Banner" here looks more like a card of some sort:

Screen Shot 2019-10-20 at 3 03 46 PM

Maybe a different name can be picked to save "Banner" for the things in #688?

**Comment by mmbotelho** [comment]
We can call them _Callout Cards_ instead, as in [Shopify's Polaris](https://polaris.shopify.com/components/structure/callout-card#navigation).

@sandrina-p @taoeffect

**Review by mmbotelho — CHANGES_REQUESTED** [review]
- The dropdowns have the wrong font - should be body text;
- The dropdown list-item should be 32px high (the implemented version has 40-something px);
- This spacing:
Captura de ecrã 2019-10-20, às 15 22 30


Other than that I couldn't find anything else wrong. I am not happy at all with how it looks on mobile, but that's work for me, don't worry about it 😄

**Review by mmbotelho — DISMISSED** [review]
Great job! 🎉

File: frontend/views/containers/settings/Appearence.vue [hunk 1]
```diff
@@ -5,6 +5,14 @@
 
       SelectorTheme
 
+    section.card
+      h2.settings-subtitle Reduced Motion
+
+      label
+        input(type='checkbox' :checked='$store.state.reducedMotion' @change='handleCheckbox')
+        i18n Reduced motion
+        .help When enabled the amount of animations you see around are reduced.
```
Comment (taoeffect) [inline frontend/views/containers/settings/Appearence.vue]: missing i18n here and on line 9

**Review by taoeffect — CHANGES_REQUESTED** [review]
Minor i18n thing in appearance, once that's fixed it's GTD!

File: frontend/views/containers/settings/Appearence.vue [hunk 2]
```diff
@@ -5,6 +5,14 @@
 
       SelectorTheme
 
+    section.card
+      h2.settings-subtitle Reduced Motion
```
Comment (taoeffect) [inline frontend/views/containers/settings/Appearence.vue:9]: "and on line 9" :)

Comment (taoeffect) [reply frontend/views/containers/settings/Appearence.vue:9]: Noticing, on lines 4, 17, and 23 - 24 too

File: test/cypress/support/commands.js [hunk 3]
```diff
@@ -40,16 +40,15 @@ Cypress.Commands.add('giLogin', (userName, password = '123456789') => {
...
-  cy.getByDT('app').then(($app) => {
-    if ($app.find('[data-test="userProfile"]').length) {
-      cy.getByDT('settingsBtn').click()
-      cy.getByDT('link-logout').click()
-      cy.getByDT('closeModal').should('not.exist')
-    } else {
-      cy.getByDT('logout').click()
-    }
-  })
+Cypress.Commands.add('giLogout', ({ hasNoGroup = false } = {}) => {
```
Comment (taoeffect) [inline test/cypress/support/commands.js:43]: this doesn't seem like valid javascript?

**Comment by taoeffect** [comment]
**NOTE: this is an exceptional merge that violates Rule 4 of our contributions guidelines because it's during a hackathon and I have approval from @sandrina-p to merge it, and @sandrina-p cannot merge it herself because the build is failing, but it's an error unrelated to this PR.**

**Pull request merged**